src/app/[locale]/not-found.tsx

```
@@ -0,0 +1,31 @@
+  
+  return (
+    <div className='flex min-h-screen items-center justify-center px-4 sm:px-6 lg:px-8'>
+      <div className='w-full max-w-md space-y-8 text-center'>
+        <div>
+          <h1 className='text-9xl font-bold text-gray-200 dark:text-gray-700'>404</h1>
+          <h2 className='mt-4 text-2xl font-bold text-gray-900 dark:text-gray-100'>
+            {t('pageNotFound')}
+          </h2>
+          <p className='mt-2 text-gray-600 dark:text-gray-400'>
+            {t('pageNotFoundDescription')}
+          </p>
```

ESLint

[error] Replace `⏎············{t('pageNotFoundDescription')}⏎··········` with `{t('pageNotFoundDescription')}`

(prettier/prettier)